### PR TITLE
doc: replace mdbook with docsify

### PR DESCRIPTION
**Pull request by skaldarnar**
As discussed on Discord in `#documentation` and in the org meeting I'm going for https://docsify.js.org as a simple solution for serving our documentation. 

Therefore, this PR migrates the little docs we have for TeraNUI to be served client-side by _docsify_ (instead of [mdbook](https://rust-lang.github.io/mdBook/)). As _docsify_ does not require any generation step the Github workflow to build and publish `gh-pages` is obsolete.

To try this out, simply start a web server in the `docs/` folder or run `docsify serve docs/` (if you installed the JS binary).

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <title>TeraNUI</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+  <meta name="description" content="A Guide to Terasology's UI framework.">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/lib/themes/vue.css">
+  <link rel="stylesheet" href="_assets/styles.css" />
+</head>
+<body>
+  <div id="app"></div>
+  <script src="//cdn.jsdelivr.net/npm/docsify-edit-on-github"></script>
```
Comment (keturn) [inline docs/index.html:14]: thought: I was wondering why all these `<script>` are in the body, not the document head.

and I'm vaguely remembering something about it sometimes being good to put scripts at the _end_ of the document for some reason, which I guess is what this is doing. It's just that there's (almost) nothing in the body before the end!

**Review by keturn — COMMENTED** [review]
looks fine code-wise. 👍 

I haven't tried `docsify serve` yet.

File: docs/index.html (hunk 1 above)
Comment (skaldarnar) [reply docs/index.html:14]: docsify will fill in the `app` with content. I did not question the choice where to put the script tags, but all examples seems to agree that you put the includes for docsify and plugins at the end of the page :shrug:

**Review by jdrueckert — APPROVED** [review]
`docsify serve` tested out fine

**Pull request merged**